src/modules/mavlink/mavlink_receiver.cpp

```diff
@@ -1356,11 +1351,21 @@ MavlinkReceiver::handle_message_odometry(mavlink_message_t *msg)
 		PX4_ERR("Body frame %u not supported. Unable to publish velocity", odom.child_frame_id);
 	}
 
-	if (odom.frame_id == MAV_FRAME_LOCAL_FRD) {
-		_visual_odometry_pub.publish(odometry);
+	/**
+	 * Supported local frame of reference is MAV_FRAME_LOCAL_NED while the
+	 * supported sources of the data/tesimator type are MAV_ESTIMATOR_TYPE_VISION,
+	 * MAV_ESTIMATOR_TYPE_VIO and MAV_ESTIMATOR_TYPE_MOCAP
+	 */
+	if (odom.frame_id == MAV_FRAME_LOCAL_NED) {
```

Comment:
I prefer the LOCAL_FRD frame for this. Since the vision parent frame is for sure not aligned with north.

Reply:
`MAV_FRAME_LOCAL_FRD` was deprecated.

Reply:
Can we support both?
```c++
	if (odom.frame_id == MAV_FRAME_LOCAL_NED || odom.frame_id == MAV_FRAME_LOCAL_FRD
) {
```

Reply:
Are you sure? https://github.com/mavlink/mavlink/blob/master/message_definitions/v1.0/common.xml#L913

Reply:
I am just asking because in Gazebo we are still using `MAV_FRAME_LOCAL_NED`.

Reply:
> Are you sure? https://github.com/mavlink/mavlink/blob/master/message_definitions/v1.0/common.xml#L913

Sorry I fixed that after a quick look.

Reply:
You can add both. In the end you still need to configure the EKF_AID_MASK correctly

Reply:
> You can add both. In the end you still need to configure the EKF_AID_MASK correctly

OK I guess I can add a note as a comment regarding that.